modules/trusted-firmware-m/Kconfig.psa.defconfig

```diff
@@ -67,7 +67,7 @@ config PSA_WANT_ALG_RIPEMD160
 	default n
 config PSA_WANT_ALG_MD5
 	bool
-	default y
+	default n
```

Comment:
Note: TCP module used legacy MD5 APIs when CONFIG_NET_TCP_ISN_RFC6528 is enabled. 

We currently convert PSA_WANT_ALG_MD5 to MBEDTLS_MD5_C in the file to convert PSA_WANT_ALG_XXXX to builtin equivalents. Please see: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/nrf_security/configs/nrf-config.h#L47

I recommend changing this to `default y if !NET_TCP_ISN_RFC6528`

Reply:
It would be impossible to satisfy this condition with CONFIG_PSA_CORE_OBERON.
We could have:
```
config PSA_WANT_ALG_MD5
      bool
      depends on PSA_CORE_BUILTIN
      default y if NET_TCP_ISN_RFC6528
```

Reply:
Given that it appears CONFIG_NET_TCP_ISN_RFC6528  is using legacy APIs and do not support PSA I don't think we need to enable this.

Reply:
We have multiple ways of solving this if it becomes a problem in the future. The proper solution is to not use MD5 for this in Zephyr. We should focus our attention on that.